pkgs/development/libraries/openssl/default.nix

```
@@ -192,6 +194,8 @@ let
       rm -r $etc/etc/ssl/misc
 
       rmdir $etc/etc/ssl/{certs,private}
+
+      ${lib.optionalString (conf != null) "cat ${conf} > $etc/etc/ssl/openssl.cnf"}
     '';
```

```suggestion
    '' + lib.optionalString (conf != null) ''
      cat ${conf} > $etc/etc/ssl/openssl.cnf
    '';
```
Normally not a big fan of this but shouldn't this cut down the amount of rebuilds?

---

Probably, but I'd guess not significantly, since this PR changes `openldap` and `python3`.